src/endpoints/getGif.ts

```diff
@@ -0,0 +1,48 @@
+import { NextFunction, Request, Response } from "express";
+import fetch from "node-fetch";
+
+import debug from "../utils/debug";
+
+/**
+ * This is the get gif endpoint, this endpoint will return the gif images from a search term,
+ * or trending is search parameter is not provided.
+ *
+ * @param {string} search The search term.
+ * @returns {Array<unknown>} Returns the gif images in an array format.
+ * @see https://developers.giphy.com/docs/api/schema#gif-object for schema def.
```

Comment:
Change ```def ``` to ```definition``` for complete sentence/word

Reply:
Fixed in commit number:[8e6a360](https://github.com/Chillandchat/api/pull/10/commits/8e6a360584283b84aa13c4dc9c1af0e150b0b522)